### ISSUE TITLE
Wav2vec checkpoint and validation

### ISSUE THREAD

**Issue by haythemdhieb** [pytorch/fairseq#2973]
## 🐛 Bug

My model (Wav2vecCtc) is simply not loading  the validation data and is not checkpointing
![Screenshot from 2020-11-30 16-04-31](https://user-images.githubusercontent.com/44007597/100626209-cfeaa500-3325-11eb-9a2d-5265381af655.png)

#### Code sample
`
python finetune_wav2vec.py task.data=/home/haythem/Desktop/Work/NLP/arabic-assistant/wav2vec/data model.w2v_path=/home/haythem/Downloads/wav2vec_small.pt        --config-name base_1h

`
I tried to modify the config file but still not working:

common:
  fp16: true
  log_format: json
  log_interval: 200

checkpoint:
  save_interval: 1000
  save_interval_updates: 50
  keep_interval_updates: 1
  no_epoch_checkpoints: True
  save_dir: wav2vec/checkpoint/
  best_checkpoint_metric: wer

task:
  _name: audio_pretraining
  data: ???
  normalize: false
  labels: ltr

dataset:
  num_workers: 1
  max_tokens: 3200000
  batch_size: 1
  skip_invalid_size_inputs_valid_test: False
  validate_after_updates: 10
  validate_interval: 1000
  

distributed_training:
  ddp_backend: no_c10d
  distributed_world_size: 1

criterion:
  _name: ctc
  zero_infinity: true

optimization:
  max_epoch: 4
  max_update: 13000
  lr: [0.00005]
